### DISCUSSION TITLE
A bug should i report it?

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [komikku-app/komikku#207] [General]
![Screenshot_2024-07-28-11-03-16-566_app.komikku.jpg](https://github.com/user-attachments/assets/137846ec-3ac9-4fc8-949e-5433d30f04c9)

on website it is normal order

**Comment by cuong-tran** [comment]
This often happens when I restore a backup file after the series has been updated. For example, if you restore an old backup file of a few days or more, while you already open that entry on the app, then the order is mixed. Sometime you can see duplicate chapters as well.

Someone reports that it would go away after a while but I'm not sure about that.

*Temporary solution:* remove that entry from your library, then go to Advance -> Clear database -> select the source, un-check the "Keep entries with read chapters". After that, add the entry again.

The restoration process might need to be improved. But this is how upstream (Mihon) is doing.

**Reply by slavedsoul** [reply]
Hey I'm just wondering how to use tag sorting tags feature in komikku.

**Comment by HikaruIchijyo** [comment]
> This often happens when I restore a backup file after the series has been updated. For example, if you restore an old backup file of a few days or more, while you already open that entry on the app, then the order is mixed. Sometime you can see duplicate chapters as well.
> 
> Someone reports that it would go away after a while but I'm not sure about that.
> 
> *Temporary solution:* remove that entry from your library, then go to Advance -> Clear database -> select the source, un-check the "Keep entries with read chapters". After that, add the entry again.
> 
> The restoration process might need to be improved. But this is how upstream (Mihon) is doing.

yeah sometimes or aource chapter missing & they upload that chapter later

**Comment by HikaruIchijyo** [comment]
a new question:

y does sync occurs wverytime i start updating the library shouldn't it only get triggered if using trigger which is selected from option or automatically after a selected period of time?
